source/index.html.md

```diff
@@ -1385,22 +1385,88 @@ This is the main filter used when searching for startups. It contains two simple
 
 ```shell
 curl -v --cookie 'JSESSIONID=EXAMPLE_SESSION_ID' \
--X GET "https://api.netzeroinsights.com/getEnabledTags"
+-X POST "https://api.netzeroinsights.com/taxonomy/tags"
+-d '{"name": "bio", "isPlatform": true, "isFilterable": true, "isVisible": true, "offset": 0, "limit": 5}'
 ```
 
 > In case of a 200 response, the response body will contain all the available tags, with the JSON structured like the following:
      
 ```json
-[
-  {
-    "name": "3d printing",
-    "id": 1
-  },
-  {
-    "name": "advanced material",
-    "id": 2
-  }
-]
+{
+  "count": 75,
+  "results": [
+    {
+      "id": 20,
```

Comment:
in this response I'd only display:
ID
tagTypeID
label
tagFamilyLabel
tagTypeLabel

removing all the other fields (this has to be changed in the BE, the response should change based on the user role)